docs/source/common_issues.rst

```diff
@@ -218,6 +218,14 @@ daemon <mypy_daemon>`, which can speed up incremental mypy runtimes by
 a factor of 10 or more. :ref:`Remote caching <remote-cache>` can
 make cold mypy runs several times faster.
 
+Furthermore: as of `mypy 1.13 <https://mypy-lang.blogspot.com/2024/10/mypy-113-released.html>`_,
```

Comment:
This is all history, are you sure that the changelog link and the specific version reference are relevant here?

Reply:
My instinct is, we might as well include it for people locked into Mypy version X who Google "Mypy slow" to see what they get. It can always be removed from the Mypy documentation later.

Reply:
They'll try `pip install mypy[fast-cache]`, get a helpful message that such extra does not exist and, hopefully, figure out that using a version more than a year old has some drawbacks... But this is more a nitpick, I agree documentation should mention this extra and have no other objections:)